src/ClientApp/src/components/StandortForm.js

```diff
@@ -0,0 +1,325 @@
+import React, { useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { GemeindenMap } from "../GemeindenMap";
+import TextField from "@mui/material/TextField";
+import Table from "@mui/material/Table";
+import IconButton from "@mui/material/IconButton";
+import DeleteIcon from "@mui/icons-material/Delete";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import EditIcon from "@mui/icons-material/Edit";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import Typography from "@mui/material/Typography";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+import Tooltip from "@mui/material/Tooltip";
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import DetailMap from "./DetailMap";
+import ConfirmationDialog from "./ConfirmationDialog";
+
+export default function StandortForm(props) {
+  const {
+    currentStandort,
+    currentBohrung,
+    setCurrentBohrung,
+    handleNext,
+    handleClose,
+    editStandort,
+    addStandort,
+    deleteBohrung,
+  } = props;
+  const { control, handleSubmit, formState, reset } = useForm({ reValidateMode: "onChange" });
+  const { isDirty } = formState;
+
+  const [openConfirmation, setOpenConfirmation] = useState(false);
+
+  const onAddBohrung = () => {
+    let bohrung = {
+      standortId: currentStandort.id,
+      bohrprofile: [],
+      hQualitaet: 3,
+      hAblenkung: 9,
```

Comment:
REQ: Codetypen in ein Enum oder als Konstanten auslagern. Diese Codes werden u.a. auch im Bohrung-UI benötigt.